Themis/Models/Submission.swift

```diff
@@ -47,10 +48,30 @@ struct Submission: Codable {
     let results: [SubmissionResult]
 }
 
+struct ExerciseOfSubmission: Codable {
+    let problemStatement: String
+    let gradingInstructions: String
+    let gradingCriteria: [GradingCriteria]
+}
+
 struct SubmissionForAssessment: Codable {
     let id: Int
     let participation: SubmissionParticipation
-    let results: SubmissionResult
+    // let results: [SubmissionResult]
```

Comment:
Why is this commented?

Reply:
Same as above. Will work on this in the future.